ql/src/test/queries/clientpositive/udf_from_utc_timestamp.q

```diff
@@ -8,8 +8,6 @@ from_utc_timestamp('2012-02-11 04:30:00', 'PST'),
 from_utc_timestamp('2012-02-11 04:30:00', 'Europe/Moscow'),
 from_utc_timestamp('2012-02-11 04:30:00', 'GMT+8'),
 from_utc_timestamp('2012-02-11 04:30:00', 'GMT'),
-from_utc_timestamp('2012-02-11 04:30:00', ''),
-from_utc_timestamp('2012-02-11 04:30:00', '---'),
```

Comment:
was that some special case??

Reply:
This are invalid formats. Previously in such cases UTC was used, but now instead an exception is thrown:
java.time.DateTimeException: Invalid ID for ZoneOffset, invalid format

But it can not be tested here. It is tested with TestGenericUDFFromUtcTimestamp.java